doc/Administration-guide/System-installation/Cluster-setup/MariaDB-cluster-setup-guide/index.md

```diff
@@ -0,0 +1,270 @@
+```
+
+To finish, save and close the file `"/etc/mysql/my.cnf"` on all the three nodes.
+
+### Copying Debian Maintenance Configuration
+
+Currently, Ubuntu and Debian's MariaDB servers use a special maintenance user to do routine maintenance. 
+Some tasks that fall outside of the maintenance category are also executed by this user, including some important functions such as stopping MySQL.
+
+In the case of a cluster environment shared between individual nodes, the maintenance user, which randomly generates login credentials on each node, is unable to execute its commands correctly. 
+In such a case only the initial server will have the correct maintenance credentials, and the others will attempt to use their local settings to access the shared cluster environment.
+
+To fix this, copy the contents of the maintenance file to each individual node as follows:
+
+1. On one of your servers, open the Debian maintenance configuration file:
+
+   ```bash
+     sudo nano /etc/mysql/debian.cnf
+   ```
+
+   You will see a file that looks like this:
+
+   ```bash
+     [client]
+     host     = localhost
+     user     = debian-sys-maint
+     password = 03P8rdlknkXr1upf
+     socket   = /var/run/mysqld/mysqld.sock
+     [mysql_upgrade]
+     host     = localhost
+     user     = debian-sys-maint
+     password = 03P8rdlknkXr1upf
+     socket   = /var/run/mysqld/mysqld.sock
+     basedir  = /usr
+   ```
+
+   You will simply need to copy this information and paste it into the same file on each node.
+
+2. On your second and third nodes, open the same file:
+
+   ```bash
+     sudo nano /etc/mysql/debian.cnf
+   ```
+
+3. Delete the current information and paste the parameters from the first node's configuration file into these other servers' files:
+
+   ```bash
+     [client]
+     host     = localhost
+     user     = debian-sys-maint
+     password = 03P8rdlknkXr1upf
+     socket   = /var/run/mysqld/mysqld.sock
+     [mysql_upgrade]
+     host     = localhost
+     user     = debian-sys-maint
+     password = 03P8rdlknkXr1upf
+     socket   = /var/run/mysqld/mysqld.sock
+     basedir  = /usr
+   ```
+
+**They should be exactly the same now.** Save and close the files.
+
+### Start MariaDB cluster
+
+To start the MariaDB cluster, do the following:
+
+1. Stop the running MariaDB service by typing the following line on each of the nodes.
+
+   ```bash
+     sudo service mysql stop
+   ```
+
+2. Start up your first node with a special parameter.
+
+   ```bash
+     sudo service mysql start --wsrep-new-cluster
+   ```
+
+   In the cluster configuration, each node that comes online tries to connect to at least one other node specified in its configuration file to get its initial state. 
+   Without the `--wsrep-new-cluster` parameter, this command would fail because the first node is unable to connect to any other nodes.
+
+3. On each of the other nodes, start MariaDB as you normally would.
+
+   ```bash
+     sudo service mysql start
+   ```
+
```

Comment:
There is much more simple commands: "SHOW STATUS LIKE 'wsrep_cluster_size';"
Please add a note somewhere on the page that one need to open following ports between this two nodes:
4444, 4567. This is important for wsrep communication.